app/src/main/java/org/fossasia/openevent/general/auth/AuthFragment.kt

```diff
@@ -0,0 +1,116 @@
+package org.fossasia.openevent.general.auth
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.MenuItem
+import android.view.View
+import android.view.ViewGroup
+import androidx.lifecycle.Observer
+import androidx.navigation.Navigation
+import androidx.navigation.fragment.navArgs
+import kotlinx.android.synthetic.main.fragment_auth.view.getStartedButton
+import kotlinx.android.synthetic.main.fragment_auth.view.email
+import kotlinx.android.synthetic.main.fragment_auth.view.rootLayout
+import org.fossasia.openevent.general.BuildConfig
+import org.fossasia.openevent.general.PLAY_STORE_BUILD_FLAVOR
+import org.fossasia.openevent.general.R
+import org.fossasia.openevent.general.utils.Utils
+import org.fossasia.openevent.general.utils.Utils.hideSoftKeyboard
+import org.fossasia.openevent.general.utils.Utils.show
+import org.fossasia.openevent.general.utils.Utils.progressDialog
+import org.fossasia.openevent.general.utils.extensions.nonNull
+import org.jetbrains.anko.design.longSnackbar
+import org.jetbrains.anko.design.snackbar
+import org.koin.androidx.viewmodel.ext.android.sharedViewModel
+import org.koin.androidx.viewmodel.ext.android.viewModel
+
+class AuthFragment : Fragment() {
+    private lateinit var rootView: View
+    private val authViewModel by viewModel<AuthViewModel>()
+    private val safeArgs: AuthFragmentArgs by navArgs()
+    private val smartAuthViewModel by sharedViewModel<SmartAuthViewModel>()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        if (BuildConfig.FLAVOR == PLAY_STORE_BUILD_FLAVOR) {
+            smartAuthViewModel.requestCredentials(SmartAuthUtil.getCredentialsClient(requireActivity()))
+            smartAuthViewModel.isCredentialStored
+                .nonNull()
+                .observe(this, Observer {
+                    if (it) redirectToLogin()
```

Comment:
@nikit19 Here no email address is passing, so set its default value to an empty string.